The return value of fgetc() must be stored in an int if it is compared
against EOF.

Index: src/rip.c
--- src/rip.c.orig
+++ src/rip.c
@@ -27,7 +27,7 @@
 #elif defined (HAVE_SYS_VFS_H)
 #include <sys/vfs.h>
 #endif
-#if defined(__FreeBSD__) || defined(__NetBSD__)
+#if defined(__FreeBSD__) || defined(__NetBSD__) || defined(__OpenBSD__)
 #include <sys/param.h>
 #include <sys/mount.h>
 #endif
@@ -663,7 +663,7 @@ void RemoveM3U(GripInfo *ginfo)
 {
   FILE *fp;
   EncodeTrack enc_track;
-  char testchar;
+  int testchar;
   GString *str;
 
   if (!ginfo->have_disc)
